api/queries_issue.go

```diff
@@ -443,6 +450,44 @@ func IssueReopen(client *Client, repo ghrepo.Interface, issue Issue) error {
 	return err
 }
 
+func IssueTemplates(client *Client, repo *Repository) ([]IssueTemplate, error) {
+	type response struct {
```

Comment:
We could simplify/shorten this GQL request using the functionality provided by `github.com/shurcooL/githubv4` so that the response struct also acts as the query. Many of the queries in `api/queries_repo.go` use this style. `MilestoneByNumber` is a short simple one that might be good to copy from.